src/log.cr

```diff
@@ -3,8 +3,9 @@
 # The messages, or `Log::Entry` have associated levels, such as `Info` or `Error`
 # that indicate their importance. See `Log::Severity`.
 #
-# To log a message `trace`, `debug`, `info`, `notice`, `warn`, `error`, and `fatal` methods
-# can be used. They expect a block that will evaluate to the message of the entry.
+# To log a message use the `#trace`, `#debug`, `#info`, `#notice`, `#warn`,
+# `#error`, and `#fatal` methods. They expect a block that will evaluate to the
+# message of the entry:
```

Comment:
I switched this from:

"[methods] can be used" to "use the [methods]" which feels like it is more consistent with the technical writing tone I've observed elsewhere in the docs.